tests/test_learning.py

```diff
@@ -165,6 +165,13 @@ def test_decision_tree_learner():
     assert dTL([7.5, 4, 6, 2]) == "virginica"
 
 
+def test_information_content():
+    assert information_content([]) == 0
+    assert information_content([4]) == 0
+    assert information_content([5, 4, 0, 2, 5, 0]) == 1.9237949406953985
+    assert information_content([1.5, 2.5]) == 0.9544340029249649
```

Comment:
Can you do `assert information_content([1.5, 2.5]) > 0.9`

Reply:
Sure. I am on it.